lib/workflows/provision.js

```diff
@@ -150,7 +150,7 @@ function preparePayload(job, cb) {
         params['kernel_version'] = payload.image.tags.kernel_version;
     }
 
-    if (payload.image.type === 'lx-dataset') {
+    if (payload.image.type === 'lx-dataset' || payload.image.type === 'lxd') {
         params['brand'] = 'lx';
```

Comment:
Wait... so containers on a Linux CN are treated like LX Brand zones on SmartOS?

Reply:
Yes - they use the same brand marker (to mean Linux container) - the original idea (before lxd was chosen) was that we would share the same images between linux and smartos-lx (this didn't pan out so well and we subsequently chose lxd as the image format for linux containers).